recipes/gw/conda_build_config.yaml

```diff
@@ -1,4 +1,12 @@
 MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
   - "10.15"                # [osx and x86_64]
+
 MACOSX_SDK_VERSION:        # [osx and x86_64]
   - "10.15"                # [osx and x86_64]
+
+CFLAGS:
+  - "-mmacosx-version-min=10.15"  # [osx and x86_64]
```

Comment:
All these new entries are not really used because of the `skip` in meta.yaml - https://github.com/bioconda/bioconda-recipes/pull/51426/files#diff-fe69e329f31bbed1cb39dcc0a94882f3501220a7ca03695a50000f1390c98cf3R34

Reply:
Ah yes, however I would like to try and get osx x64 working again in the future, so left it in for now.